### PR TITLE
Tables MVP

### LINKED ISSUE #67: Table (MVP)
- [x] Inventory
- [x] Research
- [x] Explore
- [x] Create
- [x] Document
- [ ] Release

**Figma File:**
https://www.figma.com/file/ZKo95ME2djrr6XUakP7Yw9/Tables?node-id=0%3A1

**Reference JIRA Epic:**
https://oktainc.atlassian.net/browse/UX-1717?jql=component%20%3D%20%22Design%20System%22%20AND%20status%20!%3D%20Done

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
This adds documentation and styling for `<table> / .ods-table`, as well as docs and styling for indeterminate checkboxes.

Closes #67

### CODE REVIEW

File: packages/odyssey/src/scss/components/_table.scss [hunk 1]
```diff
@@ -26,19 +26,15 @@
   }
 
   caption {
-    margin-bottom: $small-spacing;
-    color: $text-heading;
-    font-size: ms(1);
-    font-weight: 600;
-    text-align: left;
+    @include is-visually-hidden;
```
Comment (arnoldsandoval-okta) [inline packages/odyssey/src/scss/components/_table.scss]: for my education: why is this different than the class name `.m-visually-hidden` shouldn't the class name and the mixin name be consistent?

File: packages/docs/source/components/table.md [hunk 2]
```diff
@@ -638,162 +699,266 @@ The <code>data-null</code> attribute will ensure the table styling is adjusted.
...
+            <td>São Tomé</td>
           </tr>
           <tr>
-            <td data-title="Person">Lijsbeth Bakker</td>
-            <td data-title="Username">lbakker@integrationcorp.com</td>
-            <td data-title="Primary Email">lbakker@workday.net</td>
-            <td data-title="Status">Password reset</td>
+            <td class="is-ods-table-checkbox">
+              <input class="ods-checkbox" type="checkbox" name="row[9]" id="checkbox-9" value="check-9">
+              <label class="ods-checkbox--label" for="checkbox-0">
```
Comment (arnoldsandoval-okta) [inline packages/docs/source/components/table.md]: When clicking on this, it toggles the first row's checkbox.

```suggestion
              <label class="ods-checkbox--label" for="checkbox-9">
```

File: packages/docs/source/components/table.md [hunk 3]
```diff
@@ -638,162 +699,266 @@ The <code>data-null</code> attribute will ensure the table styling is adjusted.
...
+            <td>Singapore</td>
           </tr>
           <tr>
-            <td data-title="Person">Chloé Ballantyne</td>
-            <td data-title="Username">cballantyne@integrationcorp.com</td>
-            <td data-title="Primary Email">cballantyne@workday.net</td>
-            <td data-title="Status">Password reset</td>
+            <td class="is-ods-table-checkbox">
+              <input class="ods-checkbox" type="checkbox" name="row[10]" id="checkbox-10" value="check-10">
+              <label class="ods-checkbox--label" for="checkbox-0">
```
Comment (arnoldsandoval-okta) [inline packages/docs/source/components/table.md]: Same as above:

```suggestion
              <label class="ods-checkbox--label" for="checkbox-10">
```

File: packages/odyssey/src/scss/components/_table.scss (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/odyssey/src/scss/components/_table.scss]: ^ unintentional. it's a holdover from our original codebase - we should definitely set some better standards around naming.

**Pull request merged**